nservicebus/azure-service-bus/transaction-support.md

```diff
@@ -7,37 +7,66 @@ tags:
+* It requires the use of the `Serializable` isolation level when it's used inside a transaction scope (the most restrictive isolation level that does not permit `dirty reads`, `phantom reads` and `non repeatable reads`; will block any reader until the writer is committed). For more information refer to [Transaction Isolation Levels Explained in Details](http://dotnetspeak.com/2013/04/transaction-isolation-levels-explained-in-details) article. NServiceBus Azure Service Bus transport configuration is therefore hard-coded to `Serializable` isolation level. Users can't override it.
+* It also proactively checks to see whether it is used alone inside a transaction and it will throw an exception if used together with another transactional resource such as a database connection. To allow the user code to still use a transaction for it's database connections, in another transaction scope provided by NServiceBus around a handler, the 2 scopes are kept apart and synchronized using an implementation of `IEnlistmentNotification` for both the send and completion operations. (Schematically represented by the orange to yellow star notation in the diagram)
+* Finally, it does not allow send operation on different messaging entities in the same transaction. So for example a send to Queue1 and a send to Queue2 in the same scope, will throw an exception. Again this problem is negated using separate instances of `IEnlistmentNotification` on each send operation. 
+
+The benefit of this approach is that user code can use database connections and send messages to multiple endpoints while maintaining the aspects expected from a transaction. However the downside is that each operation is individual. Even though they are executed at the same moment in a transaction, it can happen in theory that one of the operations fails, while the others do not. This problem is again mitigated by using multiple layers of retry behavior to reduce the chance, so a short interruption in connectivity or broker outage will not have a significant impact and operations will eventually succeed. However a full outage right after the database transaction committed could lead to a send operation that did not execute. 
+
+The maximum guarantee that the transport can deliver in these versions is in effect `ReceiveOnly`. 
+
+Enabling/Disabling DTC will have no effect, this setting is ignored either way as Azure Service Bus does not support it.
+
+Disabling transactions will also turn off the `PeekLock` mechanism, so that the transport immediately completes any incoming message before processing and shows no retry behavior. 
+  
+
+### Versions 7 and above
+
+In NServiceBus version 6 and above, the shape of the pipeline has changed so that a transaction scope is no longer mandatory in order to orchestrate dispatching of messages with complete/rollback behavior. The new architecture is schematically represented like this:
```

Comment:
I think this paragraph could make a better job in explaining that the transport is now in full charge / control. The core no longer tightly controls the receive process.